android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -238,7 +238,7 @@ synchronized public void setNowPlaying(ReadableMap metadata) {
         }
 
         nb.setContentTitle(title);
-        nb.setContentText(artist);
+        nb.setContentText(artist); ContextCompat.startForegroundService(context, new Intent(context, MusicControlNotification.NotificationService.class));
```

Comment:
I think this would lead to the same issue described before

Reply:
it's work fine for me with no issues

Reply:
For Android Oreo too?

Reply:
yep

Reply:
Can you explain why do you need to add this here?